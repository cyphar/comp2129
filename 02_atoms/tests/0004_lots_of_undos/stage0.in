START 5 10 10
UNDO
UNDO
PLACE 3 2
UNDO
PLACE 2 1
PLACE 0 0
PLACE 1 1
UNDO
PLACE 4 4
PLACE 1 1
PLACE 2 2
PLACE 9 2
PLACE 7 7
PLACE 2 1
PLACE 8 8
UNDO
UNDO
