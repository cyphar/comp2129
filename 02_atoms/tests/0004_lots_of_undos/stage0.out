Game Ready
Red's Turn

Cannot Undo

Cannot Undo

Green's Turn

Red's Turn

Green's Turn

Purple's Turn

Blue's Turn

Purple's Turn

Blue's Turn

Yellow's Turn

Red's Turn

Green's Turn

Purple's Turn

Cannot Place Atom Here

Blue's Turn

Purple's Turn

Green's Turn

